### PR TITLE
qodem 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by atdt**
* [x]  Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
* [x]  Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
* [x]  Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
* [x]  Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
* [x]  Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---------------------

New formula for Qodem, a telecommunications program / terminal emulator / telnet / BBS client, etc.

### CODE REVIEW

**Comment by atdt** [comment]
I updated the description but the bot still flags this as "needs response". Maybe this comment will get it unstuck?

File: Formula/qodem.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                          "--disable-sdl",
+                          "--disable-serial",
+                          "--disable-upnp",
+                          "--disable-x11"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/qodem", "--exit-on-completion", "--capfile", testpath/"qodem.out", "uname"
+    assert_match(/Darwin/, File.read(testpath/"qodem.out"))
```
Comment (SMillerDev) [inline Formula/qodem.rb:24]:
```suggestion
    assert_match("Darwin", File.read(testpath/"qodem.out"))
```

Comment (atdt) [reply Formula/qodem.rb]: Thanks. I took the suggestion and omitted the parentheses, to match the style of the previous line.

**Review by SMillerDev — APPROVED** [review]
Thanks @atdt  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Review by nandahkrishna — APPROVED** [review]

**Pull request closed without merging**